build.gradle

```diff
@@ -617,6 +617,28 @@ task htmlValidate(type: Exec, group: 'Format') {
     ]
 }
 
+def javadocDirs = [
+    project(':checker-util').sourceSets.main.allJava,
+    project(':checker-util').sourceSets.test.allJava,
+    project(':checker-qual').sourceSets.main.allJava,
+    project(':checker').sourceSets.main.allJava,
+    project(':checker').sourceSets.test.allJava,
+    project(':framework').sourceSets.main.allJava,
+    project(':framework').sourceSets.test.allJava,
+    project(':dataflow').sourceSets.main.allJava,
+    project(':dataflow').sourceSets.test.allJava,
+    project(':javacutil').sourceSets.main.allJava,
+    project(':framework-test').sourceSets.main.allJava,
+    project(':framework-test').sourceSets.test.allJava
+]
+
+project(':checker').afterEvaluate {
+    javadocDirs += project(':checker').sourceSets.testannotations.allJava
```

Comment:
Should the testannotations appear in the main javadoc?
Or should there be javadoc generated for the "main" classes and requirejavadoc for the main classes plus the testannotations?

Reply:
Add a different variable for `requireJavadoc` task